guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,159 @@
+[id="Upgrading-Project-from-EL7-to-EL8{context}"]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+LEAPP is the supported tool for {RHEL} 7 to {RHEL} 8 upgrades.
+
+.Prerequisites
+* {Project} {ProjectVersionPrevious} running on CentOS 7.
+- Upgrading does not work on {RHEL}. You must have a {RHEL}-compatible LEAPP build to upgrade.
+* Access to a local repository server mirroring all required repositories.
+* No outstanding updates.
+* You can upgrade a CentOS7 system before you upgrade {Project}.
```

Comment:
Please follow conventions when referring to an operating system. This is not how it is written.

Reply:
Some of the other writers mentioned that they have used CentOS7 when documenting. What is the proper naming convention for an operating system?

Reply:
If you google and find the CentOS 7 documentation or website, you'll find how to write it.

Reply:
Brian, I am sorry. I think that I came across quite short here in retrospect and it wasn't my intention. All I meant is that there are certain ways that all operating systems, projects, third party infrastructure etc can and should be referred to specifically and it's easy enough to find them. Their project or product websites are the authority on how it should be spelled rather than myself or any writer.